### PR TITLE
Updated to 0.6.6

### PR DESCRIPTION
**Pull request by derekcollison**

### CODE REVIEW

File: library/nats [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # maintainer: Derek Collison <derek@apcera.com> (@derekcollison)
 # maintainer: Waldemar Salinas <wally@apcera.com> (@wallyqs)
 
+0.6.6: git://github.com/nats-io/nats-docker@9909ce47584df326aa0089e4a1f71af89e5baea6
 0.6.4: git://github.com/nats-io/nats-docker@59e0d88a686f727b15dda47db5323c3722e6d85a
```
Comment (tianon) [inline library/nats]: Is there a reason to keep this `0.6.4` reference around?  We won't delete the already-built tag from the Hub, but removing it here will remove it from the `Supported` list at the top of the image description.

**Comment by tianon** [comment]
Other than 0.6.4 still being listed, LGTM (https://github.com/nats-io/nats-docker/compare/59e0d88a686f727b15dda47db5323c3722e6d85a...9909ce47584df326aa0089e4a1f71af89e5baea6)

**Comment by derekcollison** [comment]
I can remove it if that is what you prefer..

**Comment by yosifkit** [comment]
@derekcollison, it is up to you. If it is not going to get updates, I would suggest dropping it so that you don't get users wanting fixes on it.

**Comment by derekcollison** [comment]
ok pushed update..

**Comment by tianon** [comment]
LGTM :+1:

Build test of #990; 23b9f10e36a5a4aca489307c83dc057bd3f09df6 (`nats`):

```console
$ url="https://raw.githubusercontent.com/docker-library/official-images/23b9f10e36a5a4aca489307c83dc057bd3f09df6/library/nats"
$ bashbrew build "$url"
Cloning nats (git://github.com/nats-io/nats-docker) ...
Processing nats:0.6.6 ...
Processing nats:latest ...
$ bashbrew list --uniq "$url" | xargs test/run.sh
testing nats:0.6.6
	'cve-2014--shellshock' [1/1]...passed
```

**Comment by yosifkit** [comment]
LGTM

**Pull request merged**